### PR TITLE
Javascript javascript1 week1

### PR DESCRIPTION
**Pull request by Hailegebriel22**

### CODE REVIEW

File: git/week1/jsweek1.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>javascript1-week1</title>
+</head>
+
+<body>
+    <h1> Home work javascript1-week1</h1>
```
Comment (cecastosic) [inline git/week1/jsweek1.html:12]: Is it an idea to include the js file here? 🙂 Or if not, why do you need HTML at all?

File: git/week1/jsweek1.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    console.log(' Julia is paying ' + Math.abs(houseCostJulia - houseExpectedPriceForJulia) + ' extra.')
+}
+else if (houseExpectedPriceForJulia == houseCostJulia) {
+    console.log(' Julia is paying  expected amount.')
+}
+else {
+    console.log(' Julia is paying ' + Math.abs(houseExpectedPriceForJulia - houseCostJulia) + ' less.')
+}
+
+
```
Comment (cecastosic) [inline git/week1/jsweek1.js:70]: Prettier 🙂

File: git/week1/jsweek1.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
+// Homework javascript1-week1
+
+//  Age-ify (A future age calculator)
+
+var yearOfBirth = 1987;
```
Comment (cecastosic) [inline git/week1/jsweek1.js:5]: You should use let or const, not var.

File: git/week1/jsweek1.js [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+var yearFuture = 2030;
+var age = yearFuture - yearOfBirth;
+console.log('You will be ' + age + ' years old in ' + yearFuture);
```
Comment (cecastosic) [inline git/week1/jsweek1.js:8]: Check Template literals to log this sentence in a more elegant way https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

File: git/week1/jsweek1.js [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+// Goodboy-Oldboy (A dog age calculator)
+
+var dogYearOfBirth = 2020
+var dogYearFuture = 2030;
+var yearDifference = dogYearFuture - dogYearOfBirth;
+var dogYear = 7 * (yearDifference);
+let shouldShowResultInDogYears = 'false';
+
+if (shouldShowResultInDogYears == 'true') {
```
Comment (cecastosic) [inline git/week1/jsweek1.js:18]: ` if (shouldShowResultInDogYears)` is enough

File: git/week1/jsweek1.js [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+var volumeInMetersForPeter = wideForPeter * deepForPeter * heightForPeter;
+const houseCostPeter = 2500000;
+function housePriceForPeter() {
+    return ((volumeInMetersForPeter * 2.5 * 1000) + (gardenSizeInM2ForPeter * 300));
+}
+var houseExpectedPriceForPeter = housePriceForPeter();
+if (houseExpectedPriceForPeter < houseCostPeter) {
+    console.log(' Peter is paying ' + Math.abs(houseCostPeter - houseExpectedPriceForPeter) + ' extra.')
+}
+else if (houseExpectedPriceForPeter == houseCostPeter) {
```
Comment (cecastosic) [inline git/week1/jsweek1.js:41]: `==` or `===`?

File: git/week1/jsweek1.js [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+    console.log(' Julia is paying ' + Math.abs(houseExpectedPriceForJulia - houseCostJulia) + ' less.')
+}
+
+
+
+
+
+// Ez Namey (Startup name generator)
+
+let firstWords = ['Unique', 'Green', 'Sunny', 'Reliable', 'Trust', 'Simple', 'Speedy', 'vision', 'Sky', 'Future']
```
Comment (cecastosic) [inline git/week1/jsweek1.js:76]: why let?

File: git/week1/jsweek1.js [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+    console.log('Your dog will be ' + dogYear + ' dog years in ' + dogYearFuture)
+}
+else {
+    console.log('Your dog will be ' + yearDifference + ' Human years in ' + dogYearFuture)
+}
+
+// Housey pricey (A house price estimator)
+// evaluation for Peter 
+
+var wideForPeter = 8;
```
Comment (cecastosic) [inline git/week1/jsweek1.js:28]: Imagine you can put all values in the same array for both Peter and Julia, for example,
`const volumeInMeters = [8*10*10,5*11*8]; ` `const gardenSizeInM2 = [100,70];`
where you know that volumeInMeters[0] is Peter's house volume, and another one is Julia's etc.
Can you refactor the rest of your code so you will have fewer lines of code?

File: git/week1/jsweek1.js [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+}
+
+
+
+
+
+// Ez Namey (Startup name generator)
+
+let firstWords = ['Unique', 'Green', 'Sunny', 'Reliable', 'Trust', 'Simple', 'Speedy', 'vision', 'Sky', 'Future']
+let secondWords = ['Systems', 'Apps', 'Company', 'Partners', 'Business', 'Associates', 'Technology', 'Consultant', 'Digital', 'Corporates']
```
Comment (cecastosic) [inline git/week1/jsweek1.js:77]: Please use `;`

File: git/week1/jsweek1.js [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+
+/* generate one random numbers for each of the name arrays, to give flexibility.
+One can choose any name from firstWords and match it with any name to secondWords.  */
+
+const randomNumber1 = Math.floor(Math.random() * 10);
+const randomNumber2 = Math.floor(Math.random() * 10);
+let startupName = firstWords[randomNumber1] + ' ' + secondWords[randomNumber2]
```
Comment (cecastosic) [inline git/week1/jsweek1.js:84]: why let?

**Review by cecastosic — COMMENTED** [review]
Good job @Hailegebriel22 👍 I added some comments that can help you with refactoring. Nice with comments, but please use Prettier for formatting. 🌸

File: git/week1/jsweek1.html (hunk 1 above)
Comment (Hailegebriel22) [reply git/week1/jsweek1.html:12]: I am hoping to see my work on the template page console, that is why I have HTML file.

File: git/week1/jsweek1.js (hunk 7 above)
Comment (Hailegebriel22) [reply git/week1/jsweek1.js:76]: const firstWords ?

File: git/week1/jsweek1.html (hunk 1 above)
Comment (cecastosic) [reply git/week1/jsweek1.html:12]: Then you are missing `<script src="jsweek1.js"></script>`

**Pull request merged**